### PR TITLE
Created Contact Form

### PR DESCRIPTION
**Pull request by wx02shi**
closing #10 
image

I don't remember how to properly set up the form to send data, but it seems like it's a part of issue 64 anyways.
I also don't remember if the "submit" button needs to be type="submit". If the form needs one, then it conflicts with the default button one of us built, as it already uses 'type' for its colour. In ss below, if the type of the button is not exactly "purpleWhite", then it will become a default bootstrap button
.image

Also changed some of the colours. Before we were using the css default "steelblue", so I changed them to the specific blue of the logo.

### CODE REVIEW

File: components/Contact.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import Title from './Title'
+import HomeMadeButton from './shared/Button'
+import { Form, Col } from 'react-bootstrap'
+
+import styles from '../styles/Contact.module.css'
+
+const Contact = () => {
```
Comment (functionstackx) [inline components/Contact.js:8]: in this form, where do I paste my rest API URL to send the data to?

Comment (wx02shi) [reply components/Contact.js:8]: good question

Comment (wx02shi) [reply components/Contact.js:8]: I don't know how forms work

Comment (LemonFace0309) [reply components/Contact.js:8]: lol

**Review by LemonFace0309 — APPROVED** [review]

**Pull request merged**